pkg/apis/autoscaling/types.go

```diff
@@ -119,17 +130,25 @@ type ScalingPolicySelect string
 
 const (
 	// MaxPolicySelect selects the policy with the highest possible change
-	MaxPolicySelect ScalingPolicySelect = "max"
+	MaxPolicySelect ScalingPolicySelect = "Max"
 	// MinPolicySelect selects the policy with the lowest possible change
-	MinPolicySelect ScalingPolicySelect = "min"
+	MinPolicySelect ScalingPolicySelect = "Min"
 	// DisabledPolicySelect disables the scaling in this direction
-	DisabledPolicySelect ScalingPolicySelect = "disabled"
+	DisabledPolicySelect ScalingPolicySelect = "Disabled"
 )
 
-// HPAScalingRules configures the scaling policy and the policy selector
+// HPAScalingRules configures the scaling behavior for one direction.
+// This Rules are applied after calculating DesiredReplicas from metrics for the HPA.
+// They can limit the scaling velocity by specifying scaling policies.
+// They can prevent flapping by specifying the stabilization window, so that the
+// number of replicas is not set instantly, instead, the safest value from the stabilization
```

Comment:
reword -> `so that the number of replicas is not changed instantly when the metrics change but the safest value..`